### ISSUE TITLE
 Update hasher after the hoverhaul crypto

### ISSUE THREAD

**Issue by thiolliere** [polkadot-js/apps#817]
not sure it is the good place and you are probably aware of it but with substrate master @gautamdhameja and me realised when running `cargo run -- --dev` in substrate all account in UI doesn't have currency anymore, it seems because hasher has change in hoverhaul crypto PR.

indeed those following address have tokens:
```
Alice//stash: 5HYCra4YvoRbrej2TAP3StQTFrtsjtgpHbAi625TWWevdfcp
Alice: 5CfbEZ9EvFsdZ2yHPt5BsSsGpM9mZPkahsZm6WeZE87mJKKT
```

**Comment by jacogr** [comment]
PR - https://github.com/polkadot-js/apps/pull/813

However, I'm suspecting the hard derivation is broken, so depending on what we do to fix subkey, it may need additional updates. (`Alice`, `Bob` and the rest are all hard derivation). TL;DR - The above linked branch works on latest master, however won't be merged until the results for the subkey issue is shows one way or another.

**Comment by jacogr** [comment]
Closed in #813
